cypress/integration/book_details_spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Book Club book details page user flow', () => {
```

Comment:
Looks good, we'll just need to remember to come back and refactor so the tests stay up to date!